documentation/configuration/metrics-configuration.md

```diff
@@ -8,6 +8,8 @@
 Due to limitations in event counters, `dotnet monitor` supports only **one** refresh interval when collecting metrics. This interval is used for
 Prometheus metrics, livemetrics, triggers, traces, and trigger actions that collect traces. The default interval is 5 seconds, but can be changed in configuration.
 
+[7.1+] For EventCounter providers, is possible to specify a different interval for each provider. See [Per provider intervals](#per-provider-intervals-71).
```

Comment:
Are we going to hold 7.1 until all runtime issues are resolved regarding metrics? If not, then this should be `8.0+`.

Reply:
My hope is that we can ship metrics in 7.1 as soon as we get a backport for the most egregious issue.